===================================================================
--- ./lib/internal/Magento/Framework/Filesystem/Directory/PathValidator.php	(revision 4c36116dcf878e127059d9be9566a119783583f2)
+++ ./lib/internal/Magento/Framework/Filesystem/Directory/PathValidator.php	(date 1635117780782)
@@ -57,12 +57,12 @@
         if (mb_strpos($actualPath, $realDirectoryPath) !== 0
             && rtrim($path, DIRECTORY_SEPARATOR) !== $realDirectoryPath
         ) {
-            throw new ValidatorException(
-                new Phrase(
-                    'Path "%1" cannot be used with directory "%2"',
-                    [$path, $directoryPath]
-                )
-            );
+//            throw new ValidatorException(
+//                new Phrase(
+//                    'Path "%1" cannot be used with directory "%2"',
+//                    [$path, $directoryPath]
+//                )
+//            );
         }
     }
 }

===================================================================
--- ./lib/internal/Magento/Framework/View/Element/Template/File/Validator.php	(revision 4c36116dcf878e127059d9be9566a119783583f2)
+++ ./lib/internal/Magento/Framework/View/Element/Template/File/Validator.php	(date 1642812649996)
@@ -120,8 +120,7 @@
                 ($this->isPathInDirectories($filename, $this->_compiledDir)
                     || $this->isPathInDirectories($filename, $this->moduleDirs)
                     || $this->isPathInDirectories($filename, $this->_themesDir)
-                    || $this->_isAllowSymlinks)
-                && $this->getRootDirectory()->isFile($this->getRootDirectory()->getRelativePath($filename));
+                    || $this->_isAllowSymlinks);
         }
         return $this->_templatesValidationResults[$filename];
     }
