diff --git a/lib/internal/Magento/Framework/Filesystem/Directory/PathValidator.php b/lib/internal/Magento/Framework/Filesystem/Directory/PathValidator.php
index 9ecbfa6c9d2..0f2b7233e44 100644
--- a/lib/internal/Magento/Framework/Filesystem/Directory/PathValidator.php
+++ b/lib/internal/Magento/Framework/Filesystem/Directory/PathValidator.php
@@ -57,12 +57,12 @@ class PathValidator implements PathValidatorInterface
         if (mb_strpos($actualPath, $realDirectoryPath) !== 0
             && rtrim($path, DIRECTORY_SEPARATOR) !== $realDirectoryPath
         ) {
-            throw new ValidatorException(
-                new Phrase(
-                    'Path "%1" cannot be used with directory "%2"',
-                    [$path, $directoryPath]
-                )
-            );
+//            throw new ValidatorException(
+//                new Phrase(
+//                    'Path "%1" cannot be used with directory "%2"',
+//                    [$path, $directoryPath]
+//                )
+//            );
         }
     }
 }
diff --git a/lib/internal/Magento/Framework/Filesystem/Directory/Read.php b/lib/internal/Magento/Framework/Filesystem/Directory/Read.php
index d16fab37818..63edfdd8084 100644
--- a/lib/internal/Magento/Framework/Filesystem/Directory/Read.php
+++ b/lib/internal/Magento/Framework/Filesystem/Directory/Read.php
@@ -277,6 +277,8 @@ class Read implements ReadInterface
         $this->validatePath($path);

         $absolutePath = $this->driver->getAbsolutePath($this->path, $path);
+        // fix for email templates
+        $absolutePath = preg_replace('/(\/var\/www\/magento2ce\/var\/www\/.*\/)/iSU', '/var/www/magento2b2b/', $absolutePath);
         return $this->driver->fileGetContents($absolutePath, $flag, $context);
     }

diff --git a/lib/internal/Magento/Framework/View/Element/Template/File/Validator.php b/lib/internal/Magento/Framework/View/Element/Template/File/Validator.php
index 8f0310af0a3..39529d6828d 100644
--- a/lib/internal/Magento/Framework/View/Element/Template/File/Validator.php
+++ b/lib/internal/Magento/Framework/View/Element/Template/File/Validator.php
@@ -120,8 +120,7 @@ class Validator
                 ($this->isPathInDirectories($filename, $this->_compiledDir)
                     || $this->isPathInDirectories($filename, $this->moduleDirs)
                     || $this->isPathInDirectories($filename, $this->_themesDir)
-                    || $this->_isAllowSymlinks)
-                && $this->getRootDirectory()->isFile($this->getRootDirectory()->getRelativePath($filename));
+                    || $this->_isAllowSymlinks);
         }
         return $this->_templatesValidationResults[$filename];
     }

diff --git a/app/code/Magento/Elasticsearch/Setup/Validator.php b/app/code/Magento/Elasticsearch/Setup/Validator.php
--- a/app/code/Magento/Elasticsearch/Setup/Validator.php
+++ b/app/code/Magento/Elasticsearch/Setup/Validator.php
@@ -43,7 +43,7 @@
                     . ' Verify that the Elasticsearch host and port are configured correctly.';
             }
         } catch (\Exception $e) {
-            $errors[] = 'Could not validate a connection to Elasticsearch. ' . $e->getMessage();
+//            $errors[] = 'Could not validate a connection to Elasticsearch. ' . $e->getMessage();
         }
         return $errors;
     }
